spec/search_food_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Search' do
+  describe 'happy path' do
+    it 'allows users to search for food' do
+      visit root_path
+
+      fill_in :q,	with: 'sweet potatoes'
+
+      click_button 'Search'
+
+      expect(current_path).to eq(foods_path)
+
+      expect(page).to have_content('Total Results:')
+      expect(page).to have_selector('li', count: 40)
+      expect(page).to have_content('GTIN/UPC code')
+      expect(page).to have_content('Description')
+      expect(page).to have_content('Brand Owner')
+      expect(page).to have_content('Ingredients')
+    end
+  end
```

Comment:
Looks like you're on your way and just ran out of time. I'm glad to see that you got the functionality implemented and were able to write a decent feature test. Now, time to focus on the refactoring pattern, which will be key to passing the final. I recommend taking some time to refactor this in to the Facade/Poro/Service pattern you all learned and writing unit tests for each class you create. That will be paramount to passing the mod.